apps/prairielearn/elements/pl-excalidraw/pl-excalidraw.py

```diff
@@ -0,0 +1,100 @@
+import base64
+import json
+from pathlib import Path
+
+import chevron
+import lxml.html
+import prairielearn as pl
+
+
+def prepare(element_html: str, _: pl.QuestionData) -> None:
+    element = lxml.html.fragment_fromstring(element_html)
+    pl.check_attribs(element, ["name"], ["width", "height", "file", "file_dir"])
+
+
+def render(element_html: str, data: pl.QuestionData) -> str:
+    element = lxml.html.fragment_fromstring(element_html)
+    with open("pl-excalidraw.mustache", "r", encoding="utf-8") as template:
+        drawing_name = pl.get_string_attrib(element, "name")
+        initial_content: str = ""
+
+        def load_file_content() -> str:
+            file_dir = pl.get_string_attrib(
+                element, "file_dir", "client_files_question_path"
+            )
+            if file_dir not in data["options"]:
+                path_suffixes = [
+                    key for key in data["options"].keys() if key.endswith("_path")
+                ]
+                raise RuntimeError(f"{file_dir=} not a valid key ({path_suffixes})")
+            file = Path(data["options"][file_dir]) / pl.get_string_attrib(
+                element, "file"
+            )
+            if not file.exists():
+                raise RuntimeError(
+                    f"Drawing named {drawing_name} at {file} cannot be found"
+                )
```

Comment:
Nit, this matches similar errors from other elements where we check for file existence.

```suggestion
                raise RuntimeError(f"Unknown file path: {file}")
```